content/en/docs/components/katib/reference/experiment-cr.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Katib Experiment Lifecycle"
+description = "What happens after an `Experiment` is created"
+weight = 10
++++
+
+## Katib Experiment Lifecycle
+
+When user creates an Experiment, Katib Experiment controller,
+`Suggestion` controller and `Trial` controller is working together to achieve
+hyperparameters tuning for user's Machine learning model. The Experiment
+workflow looks as follows:
+
+<center>
+<img width="100%" src="/docs/components/katib/images/katib-workflow.png" alt="Katib Workflow"
+</center>
```

Comment:
You don't need to put this image in the centre, as you can see all text has been centralized: https://deploy-preview-3906--competent-brattain-de2d6d.netlify.app/docs/components/katib/reference/experiment-cr/#:~:text=The%20Experiment%20is%20submitted%20to%20the%20Kubernetes

Just use the same tags as here for image: https://github.com/kubeflow/website/blame/0d213e8b11efec11b128fe7e4c4832156b252c47/content/en/docs/components/katib/reference/architecture.md#L33-L35

Reply:
Thanks.